src/odemis/acq/fastem.py

```diff
@@ -167,6 +167,75 @@ def __init__(self, name, coordinates):
         # if None -> acquire, if not None, don't acquire again
 
 
+SCANNER_CONFIG_OVERVIEW = {
+    "multibeam_mode": False,
+    "external": False,
+    "blanker": None,  # automatic
+    "immersion": False,  # disable to get a larger field of view
+    "horizontal_fov": 1.5e-3,
+    "resolution": (1024, 884),  # px
+}
+
+
+SCANNER_CONFIG_MEGAFIELD = {
+    "multibeam_mode": True,
+    "external": False,
+    "blanker": False,
+    "immersion": True,
+    "horizontal_fov": 2.2e-5,
+    "resolution": (1024, 884),
+}
+
+
+SCANNER_CONFIG_LIVESTREAM = {
+    "multibeam_mode": False,
+    "external": False,
+    "blanker": None,  # automatic
+    "immersion": True,
+    "horizontal_fov": 2.2e-5,
+    "resolution": (1024, 884),
```

Comment:
Add a comment saying that resolution for megafield imaging is controlled by the acquisition server module (ASM).